connection.go

```diff
@@ -14,11 +14,17 @@ type Connection interface {
 	Remote() *Peer
 
 	getLocal() *Peer
-	breakTie(Connection) connectionTieBreak
 	remoteTCPAddress() string
 	isOutbound() bool
 	isEstablished() bool
+}
+
+type LConnection interface {
```

Comment:
Neither of those describe what `LConnection` is for. It does represent a connection which our peer has. `ourConnection`?

It looks `LocalConnection` would not need to exported either if it implemented an interface with an `OverlayConn` method, so the likes of https://github.com/weaveworks/weave/blob/master/router/network_router.go#L189 can use that instead of a cast to `LocalConnection`.